docs/specs/metadata.yml

```diff
@@ -28,8 +28,113 @@ inputs:
     # Title
 outputs:
   docs/a.json: |
-    { "ms.custom-metadata": null, "author":"xxx", "uid": null, "breadcrumb_path": "/breadcrumb_path", "title": "Title"}
+    { 
+      "ms.custom-metadata": null, 
+      "author":"xxx", 
+      "uid": null, 
+      "breadcrumb_path": "/breadcrumb_path", 
+      "title": "Title",    
+    }
+---
+# output _xrefmap per file to structure json
+repos:
+  https://docs.com/test#test:
+    - files:
+        docfx.yml:
+        docs/a.md: |
+          ---
+          uid: abc
+          breadcrumb_path: "/breadcrumb_path"
+          author: xxx
+          ---
+          # Title
+outputs:
+  docs/a.json: |
+    {      
+      "author":"xxx", 
+      "uid": "abc", 
+      "breadcrumb_path": "/breadcrumb_path", 
+      "title": "Title",  
+      "_xrefmap": {
+          "abc":{
+            "href": "https://docs.com/docs/a?branch=test",
+            "name": "abc",
+            "uid": "abc",
+            "specType": "uid",
+          }
+      }
+    }
+  .xrefmap.json: | 
+    {
+      "references":[
+        {"uid": "abc", "name": "abc" ,"href": "https://docs.com/docs/a?branch=test","specType": undefined },       
+      ]
+    }
+---
+# In continue build, output xrefmap to page metadata but not page data
+continueBuildOnly: true
+repos:
+  https://docs.com/test#test:
+    - files:
+        docfx.yml: |
+          outputType: pageJson
+        docs/a.md: |
+          ---
+          uid: abc
+          breadcrumb_path: "/breadcrumb_path"
+          author: xxx
+          ---
+          # Title
+outputs:  
+  docs/a.raw.page.json: | 
+    { 
+      "rawMetadata": {
+        "xrefmap": undefined
+      }
+    }
+  docs/a.mta.json: |
+    {       
+      "xrefmap": "[{\"uid\":\"abc\",\"href\":\"https://docs.com/docs/a?branch=test\",\"specType\":\"uid\",\"name\":\"abc\"}]"
+    }
+  .xrefmap.json: | 
+    {
+      "references":[
+        {"uid": "abc", "name": "abc" ,"specType": undefined },       
+      ]
+    }
 ---
+# for non-continue build, xrefmap is not output
```

Comment:
I not sure if we intentionally want this behavior difference between continue/non-continue. If we choose to expose `_xrefmap` as a standard output for both conceptual and reference, then it make sense to me to output this value both both continue and non-continue.